test/app/lib/cinematic/cinematicParser.spec.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-env jasmine */
+import { parseShot } from '../../../../app/views/play/cinematic/Command/CinematicParser'
+
+describe('parseShot', () => {
+  it('handles an empty shot', () => {
+    expect(parseShot({})).toEqual([[]])
+  })
+
+  it('parses setup correctly', () => {
+    const systems = [{
+      parseSetupShot: jasmine.createSpy().and.returnValue(['setup commands'])
+    }]
+    const shot = {
+      setupShot: 'Example setup shot'
+    }
+    const results = parseShot(shot, systems)
+    expect(systems[0].parseSetupShot).toHaveBeenCalledWith({ setupShot: 'Example setup shot' })
+    expect(results).toEqual([['setup commands']])
+  })
+
+  it('parses setup correctly with many systems', () => {
+    const systems = [{
+      parseSetupShot: jasmine.createSpy().and.returnValue(['setup command1'])
+    }, {
+      parseSetupShot: jasmine.createSpy().and.returnValue(['setup command2'])
+    }]
+    const shot = {
+      setupShot: 'Example setup shot'
+    }
+    const results = parseShot(shot, systems)
+    expect(systems[0].parseSetupShot).toHaveBeenCalledWith({ setupShot: 'Example setup shot' })
+    expect(systems[1].parseSetupShot).toHaveBeenCalledWith({ setupShot: 'Example setup shot' })
+    expect(results).toEqual([['setup command1', 'setup command2']])
+  })
+
+  it('parses nothing correctly', () => {
```

Comment:
The description of the test looks different than the test.

Reply:
I have no idea where that description came from.
It should definitely be something like `dialogNode array data passed into parseDialogNode system`.

Great pickup!